docs/install.md

```diff
@@ -69,9 +69,10 @@ _See
 for more information_
 
 1. First, login as a user with `cluster-admin` privileges. The following example
-   uses the default `system:admin` user:
+   uses the default `system:admin` user (`admin:admin` for MiniShift):
```

Comment:
are you sure it is admin:admin when installing minishift with the admin-user addons it says about system:admin instead? 

I think you can just add something like this instead if it's really needed : 

`...On Minishift the admin user to use is admin:admin`

Reply:
Relevant issue: https://github.com/minishift/minishift/issues/2107